modules/vaos/config/statsd.rb

```diff
@@ -1,6 +1,12 @@
 # frozen_string_literal: true
 
-StatsD.increment("#{VAOS::Service::STATSD_KEY_PREFIX}.get_va_appointments.total", 0)
-StatsD.increment("#{VAOS::Service::STATSD_KEY_PREFIX}.get_va_appointments.fail", 0)
-StatsD.increment("#{VAOS::Service::STATSD_KEY_PREFIX}.get_cc_appointments.total", 0)
-StatsD.increment("#{VAOS::Service::STATSD_KEY_PREFIX}.get_cc_appointments.fail", 0)
+service_methods = %w[get_appointments get_cancel_reasons get_clinic_institutions get_clinic_institutions
```

Comment:
NITPICK: might want to call this "vaos_endpoints" since this file covers a lot more than VAOS.

Reply:
Nice that it's alphabetized!